### ISSUE TITLE
catkin build fails wrong gtsam version

### ISSUE THREAD

**Issue by shabpompeiano** [MIT-SPARK/Kimera-VIO-ROS#191]
**Description:**
When following the installation guide, catkin build fails

**Command:**
```
catkin build
```

**Console output:**
```
Boost libraries missing and gtsam missing Pose3
```

Please give also the following information:
* KimeraVIO branch, tag or commit used
* GTSAM version used: 4.3
* OpenGV version used: -
* OpenCV version used: type `opencv_version` -
* Operating system and version (e.g. Ubuntu 16.04 or Windows 10):  Ubuntu 18.04
* Did you change the source code? (yes / no): no

What worked for me: 
Since Gtsam has been updated to 4.3, it seems like it is not compatible.
Checkout tag 4.1.1 of gtsam:
`cd gtsam && git checkout tags/4.1.1 `

**Comment by myboyhood** [comment]
@shabpompeiano  Yes! 4.1.1 is fine to build the project, but I find the trajectory is drifting using the default Euroc launch file
When I run `roslaunch kimera_vio_ros kimera_vio_ros_euroc.launch online:=true`
Have you change any params before running the Kimera-VIO or Kimera-VIO-ROS ?
